modules/cluster-cloud-controller-config-osp.adoc

```diff
@@ -0,0 +1,328 @@
+
+//  This option is currently a workaround for the issue https://github.com/kubernetes/ingress-nginx/issues/3996, should be removed or refactored after the Kubernetes [KEP-1860](https://github.com/kubernetes/enhancements/tree/master/keps/sig-network/1860-kube-proxy-IP-node-binding) is implemented.
+
+// | `ingress-hostname-suffix`
+// | The dns suffix to the load balancer IP address when using proxy protocol. Default nip.io
+
+//  This option is currently a workaround for the issue https://github.com/kubernetes/ingress-nginx/issues/3996, should be removed or refactored after the Kubernetes [KEP-1860](https://github.com/kubernetes/enhancements/tree/master/keps/sig-network/1860-kube-proxy-IP-node-binding) is implemented.
+
+// | `default-tls-container-ref`
+// | Reference to a tls container. This option works with Octavia, when this option is set then the cloud provider will create an Octavia Listener of type TERMINATED_HTTPS for a TLS Terminated loadbalancer.
+
+//  Format for tls container ref: `https://{keymanager_host}/v1/containers/{uuid}`
+//  Check `container-store` parameter if you want to disable validation.
+
+// | `container-store`
+// | Optional. Used to specify the store of the tls-container-ref, e.g. "barbican" or "external" - other store will cause a warning log. Default value - `barbican` - existence of tls container ref would always be performed. If set to `external` format for tls container ref will not be validated.
+
+| `max-shared-lb`
+| The maximum number of Services that can share a load balancer. The default value is `2`.
+|===
+
+// [id="ccm-config-metadata-options"]
+// == Metadata options
+
+// |===
+// | Option | Description
+
+// | `search-order`
+// | This configuration key affects the way that the provider retrieves metadata that relates to the instances in which it runs. The default value of `configDrive,metadataService` results in the provider retrieving metadata that relates to the instance from, if available, the config drive first,and then the metadata service. Alternative values are:
+//  * `configDrive`: Only retrieve instance metadata from the configuration drive.
+//  * `metadataService`: Only retrieve instance metadata from the metadata service.
+//  * `metadataService,configDrive`: Retrieve instance metadata from the metadata service first if available, and then retrieve instance metadata from the configuration drive.
+// |===
+
+// ### Multi region support (alpha)
+
+// | environment variable `OS_CCM_REGIONAL` is set to `true` - allow CCM to set ProviderID with region name `${ProviderName}://${REGION}/${instance-id}`. Default: false.
+
+[id="cluster-cloud-controller-config-overrides"]
+== Options that the Operator overrides
+
+The CCM Operator overrides the following options, which you might recognize from configuring {rh-openstack}. Do not configure them yourself. They are included in this document for informational purposes only.
```

Comment:
Do you think it is worth making the "Do not..." into an admonition? Idk what the consequences are if they miss this.

Reply:
Your effort is wasted--that's about it, AFAIK. I'm okay leaving as is, as the preceding sentence says as much.